test/soca/gw/setup_workflow_exp.sh

```diff
@@ -3,65 +3,65 @@
 bindir=$1
 srcdir=$2
 
+# export env. var.
+source "${srcdir}/test/soca/gw/runtime_vars.sh" "${bindir}" "${srcdir}"
+
 # test experiment variables
-idate=2021032318
-edate=2021032418
+idate=${PDY}${cyc}
+edate=${PDY}${cyc}
```

Comment:
this is fine, but this is not equivalent to what was here before, just a comment...

Reply:
the dates were wrong.

Reply:
It doesn't look like that `setup_expt.py` does anything with the dates. `PDY` and `cyc` from the environment that we set is what's used.

Reply:
the dates are just used to set up the rocoto XML (I believe)